packages/frontend/core/src/components/affine/setting-modal/general-setting/plans/index.tsx

```diff
@@ -21,13 +21,17 @@ import * as styles from './style.css';
 const getRecurringLabel = ({
   recurring,
   t,
+  shouldLowerCase,
 }: {
   recurring: SubscriptionRecurring;
   t: ReturnType<typeof useAFFiNEI18N>;
+  shouldLowerCase?: boolean;
```

Comment:
Use upperFirst from lodash to change this logic, let lower be default.

Reply:
Will it be better to implement with css only instead?

Reply:
oh, if css can resolve it, that is better, i don't know it.